setup.py

```diff
@@ -259,6 +259,7 @@ def compile_in(self, source_folder, build_folder, install_folder):
         # Finally, add the source folder path
         args.append(str(build_to_source_folder))
 
+        print("Running {}".format(" ".join(args)))
```

Comment:
Can we live without this line or did you need this line often during debugging? (I can leave it in if it was needed so often that you'd rather have it in the script).

Reply:
No, we can remove it. I just added it for debugging, but I didn't thought it hurt to be explicit about it.